romancal/resample/resample.py

```diff
@@ -759,3 +763,79 @@ def gwcs_into_l3(model, wcsinfo):
     # l3_wcsinfo.dec_corn4 = from bounding box
     # l3_wcsinfo.orientat = ???
     # l3_wcsinfo.orientat_local = ???
+
+
+def populate_mosaic_basic(
+    output_model: datamodels.MosaicModel, input_models: [List, ModelContainer]
+):
+    """
+    Populate basic metadata fields in the output mosaic model based on input models.
+
+    Parameters
+    ----------
+    output_model : MosaicModel
+        Object to populate with basic metadata.
+    input_models : [List, ModelContainer]
+        List of input data models from which to extract the metadata.
+        ModelContainer is also supported.
+
+    Returns
+    -------
+    None
+    """
+
+    input_meta = [datamodel.meta for datamodel in input_models]
+
+    # time data
+    output_model.meta.basic.time_first_mjd = np.min(
+        [x.exposure.start_time.mjd for x in input_meta]
+    )
+    output_model.meta.basic.time_last_mjd = np.max(
+        [x.exposure.end_time.mjd for x in input_meta]
+    )
+    output_model.meta.basic.time_mean_mjd = np.mean(
+        [x.exposure.mid_time.mjd for x in input_meta]
+    )
+    output_model.meta.basic.max_exposure_time = np.max(
+        [x.exposure.exposure_time for x in input_meta]
+    )
+    output_model.meta.basic.mean_exposure_time = np.mean(
+        [x.exposure.exposure_time for x in input_meta]
+    )
```

Comment:
I think these two are set in update_exposure_times using the exposure time image; they are supposed to be the pixel-wise mean and maxes not the exposure-wise means and maxes.  Probably these lines can just be deleted, but check if update_exposure_times is actually setting these?

Reply:
Nice catch, @schlafly! Indeed, both `basic.max_exposure_time` and `basic.mean_exposure_time` are being set in `update_exposure_times`, corresponding to the pixel-wise max and mean, respectively:

https://github.com/spacetelescope/romancal/blob/76176b06ea57b1bfe0de6d47e8f2f8187a748db4/romancal/resample/resample.py#L499-L504

I removed them from the current PR.